### PR TITLE
feat: Adding metric for sum of original_label_cost

### PR DESCRIPTION
**Pull request by peterychang**

### CODE REVIEW

File: vowpalwabbit/vw_exception.h [hunk 1]
```diff
@@ -96,6 +96,21 @@ class vw_unrecognised_option_exception : public vw_exception
   ~vw_unrecognised_option_exception() noexcept override = default;
 };
 
+class save_load_model_exception : public vw_exception
```
Comment (jackgerrits) [inline vowpalwabbit/vw_exception.h:99]: Is this meant to be here?

File: vowpalwabbit/parse_slates_example_json.h [hunk 2]
```diff
@@ -291,6 +292,8 @@ void parse_slates_example_dsjson(vw& all, v_array<example*>& examples, char* lin
       {
         assert(false);
       }
+
+      if (current_obj.HasMember("_original_label_cost")) { data->originalLabelCost = current_obj["_original_label_cost"].GetFloat(); }
```
Comment (jackgerrits) [inline vowpalwabbit/parse_slates_example_json.h:296]: You should check whether it is a float

File: vowpalwabbit/metrics.cc [hunk 3]
```diff
@@ -70,6 +70,8 @@ void list_to_json_file(dsjson_metrics* ds_metrics, std::string filename, metric_
       writer.String(ds_metrics->LastEventId.c_str());
       writer.Key("last_event_time");
       writer.String(ds_metrics->LastEventTime.c_str());
+      writer.Key("cbea_sum_cost_original");
```
Comment (lalo) [inline vowpalwabbit/metrics.cc:73]: cbea is the prefix used by cb_explore_adf, in this case this value comes from dsjson, I would rename to dsjon_sum_cost_original?

File: vowpalwabbit/vw_exception.h (hunk 1 above)
Comment (peterychang) [reply vowpalwabbit/vw_exception.h:99]: no, it is not haha

**Review by jackgerrits — APPROVED** [review]

**Pull request merged**